bin/alluxio-start.sh

```diff
@@ -184,65 +201,74 @@ done
 shift $((OPTIND-1))
 
 WHAT=$1
-
 if [[ -z "${WHAT}" ]]; then
   echo "Error: no WHAT specified"
   echo -e "${USAGE}"
   exit 1
 fi
+shift
+
+
+MOPT=$1
+# Set MOPT.
+case "${WHAT}" in
+  all|worker|workers)
+    check_mount_mode ${MOPT}
+    shift
+    ;;
+  local)
+    if [[ -z ${MOPT} || ${MOPT} == "-f" ]]; then
+      MOPT="SudoMount"
+    else
+      shift
+    fi
+    check_mount_mode ${MOPT}
+    ;;
+  *)
+    MOPT=""
+    ;;
+esac
 
 # get environment
 get_env
 
 # ensure log/data dirs
 ensure_dirs
 
```

Comment:
I feel that replacing the following `$1` with `${FORMAT}` will improve readability.